superset/assets/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -148,9 +191,14 @@ class ChartHolder extends React.Component {
           >
             <div
               ref={dragSourceRef}
-              className="dashboard-component dashboard-component-chart-holder"
+              className={`dashboard-component dashboard-component-chart-holder ${
+                inFocus ? 'inFocus' : ''
+              }`}
             >
-              {!editMode && <AnchorLink anchorLinkId={component.id} />}
+              {!editMode && (
+                <AnchorLink anchorLinkId={component.id} inFocus={inFocus} />
+              )}
+              {inFocus && ChartHolder.renderInFocusCSS(labelName)}
```

Comment:
this seems pretty hacky... is there a reason you can't just have this css rule in the css file and just add classes to the parent elements when it's in focus? I'd encourage trying to find another way to do this

Reply:
note the render function:
```
    return (
      <style>
        {`.inFocus label[for=${labelName}] + .Select .Select-control {
                    border: 2px solid #00736a;
           }`}
      </style>
    );
```
labelName is dynamic. because in 1 filter_box, there are multiple filter field. we only focus one at at time. If use general css + add classes, i need to pass the selected field name down to filter_box component. it need to add more props for filter_box component, which is not controlled by this code base. i don't want to change so many parts if one simple solution can make same result.